solver/llbsolver/ops/exec.go

```diff
@@ -324,7 +325,7 @@ func (e *execOp) Exec(ctx context.Context, inputs []solver.Result) ([]solver.Res
 		meta.Env = append(meta.Env, proxyEnvList(e.op.Meta.ProxyEnv)...)
 	}
 
-	stdout, stderr := logs.NewLogStreams(ctx)
+	stdout, stderr := logs.NewLogStreams(ctx, os.Getenv("BUILDKIT_DEBUG_EXEC_OUTPUT") == "1")
```

Comment:
Can we just use `logrus.Debugf` and eliminate this env var?

Reply:
I think this would be too much logging even for debug. I could set it to just a constant false and require a code change if this should be used (similar to the current scheduler debugging).